package/MDAnalysis/core/groups.py

```diff
@@ -1908,6 +1908,22 @@ def select_atoms(self, sel, *othersel, **selgroups):
             sphlayer *inner radius* *outer radius* *selection*
                 Similar to sphzone, but also excludes atoms that are within
                 *inner radius* of the selection COG
+            cylayer *innerRadius* *externalRadius* *zMax* *zMin* *selection*
+                selects all atoms within a cylindric layer centered in the
+                center of geometry (COG) of a given selection,
+                e.g. ``cylayer 5 10 10 -8 protein`` selects the center of
+                geometry of protein, and creates a cylindrical layer of inner
+                radius 5, external radius 10 centered on the COG. In z, the
+                cylinder extends from 10 above the COG to 8 below. Positive
+                values for *zMin*, or negative ones for *zMax*, are allowed.
+            cyzone *externalRadius* *zMax* *zMin* *selection*
```

Comment:
Could you change the order, please?
- sphzone
- sphlayer
- **cyzone**
- **cylayer**

for consistency with `sp*`?

Reply:
Sure.

Thank you for pointing this out.